stdlib/2and3/distutils/ccompiler.pyi

```diff
@@ -19,6 +19,16 @@ def new_compiler(plat: Optional[str] = ..., compiler: Optional[str] = ...,
 def show_compilers() -> None: ...
 
 class CCompiler:
+    dry_run: int
+    force: int
+    verbose: int
```

Comment:
Nit: While technically ints, these are used as bools and I'd recommend to just mark them as such. This code most likely pre-dates the introduction of bools in Python.